pkg/db/kvstore/consulclient.go

```diff
@@ -97,7 +95,7 @@ func (c *ConsulClient) Get(ctx context.Context, key string) (*KVPair, error) {
 	deadline, _ := ctx.Deadline()
 	kv := c.consul.KV()
 	var queryOptions consulapi.QueryOptions
-	queryOptions.WaitTime = GetDuration(deadline.Second())
```

Comment:
Not the right implementation if WaitTime is the duration till when to wait for response generation